cfme/tests/control/test_actions.py

```diff
@@ -150,7 +145,8 @@ def vm(request, provider, small_template, vm_name):
             pytest.skip("{} is quite likely overloaded! Check its status!\n{}: {}".format(
                 provider.key, type(e).__name__, str(e)))
 
-    def finalize():
+    @request.addfinalizer
+    def _finalize():
```

Comment:
I like putting an underscore in front of anything that is really private and should not be touched from outside.